src/hooks/useLocalStorage.js

```diff
@@ -0,0 +1,17 @@
+import { useState } from 'react';
+
+function useLocalStorage(key, initialValue) {
+    const [storedValue, setStoredValue] = useState(() => {
+        const item = window.localStorage.getItem(key);
+        return item ? JSON.parse(item) : initialValue;
+      });
+    
+    const setValue = value => {
+      setStoredValue(value);
```

Comment:
Great job using local storage. Pop quiz: would storing a user's login information be a good use of local storage?